src/UserList.js

```diff
@@ -1,34 +1,87 @@
 import React from 'react';
-import usersData from './data/users.json';
-import User from './components/user.js';
+import User from './components/User.js';
 import './css/App.css';
+import UserStore from './stores/UserStore';
+import UserAction from './actions/UserAction';
 
 
 export default class UserList extends React.Component {
     constructor (props) {
         super(props);
 
-        this.state = {
-            wantAdd: false
-        };
+        this.state = this.getAppState();
```

Comment:
We have a lot of problems in the DevTools console. Let's fix them all.

**During the first rendering:**
![image](https://user-images.githubusercontent.com/12034551/95297848-cb54c380-0883-11eb-9a23-75bea99c9f5b.png)
1) we can make `User`'s inputs `readonly` or try to set an empty onChange handler (later we are going to add a capability to edit user's entry)
2) there should be only one DOM element with the same `id` on the html page. We should use a different identificator. Such a `name` for example.
3) it's better to set `default` block in the `switch`. We prefer using `if ...else if else` etc. 
But if we have a lot of branches `switch` became more suitable. In our case we have only one action I think `switch` is too much.

**During adding a new user:**
![image](https://user-images.githubusercontent.com/12034551/95298332-985eff80-0884-11eb-8934-235ad25268b7.png)